plugins/file-conversion/skills/file-conversion/scripts/convert.sh

```
@@ -0,0 +1,59 @@
+  exit 2
+fi
+[ -f "$IN" ] || { echo "error: input file not found: $IN" >&2; exit 2; }
+
+SIZE=$(wc -c < "$IN" | tr -d ' ')
+if [ "$SIZE" -gt 25000000 ]; then
+  echo "error: file is $((SIZE / 1048576))MB — free limit is 25MB. Get a free API key for larger files: https://changethisfile.com/docs/authentication" >&2
+  exit 3
+fi
+
+TARGET=$(printf '%s' "$TARGET" | tr '[:upper:]' '[:lower:]' | sed 's/^\.//')
+BASENAME=$(basename "$IN" | tr -d '"\\')
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)